pylxd/client.py

```diff
@@ -76,21 +76,30 @@ class _APINode:
     """An api node object."""
 
     def __init__(
-        self, api_endpoint, cert=None, verify=True, timeout=None, project=None
+        self,
+        api_endpoint,
+        cert=None,
+        verify=True,
+        timeout=None,
+        project=None,
+        session=None,
```

Comment:
it's probably worth adding a comment in the docstring mentioning that if the session is passed, `cert` and `verify` parameters are ignored